### PR TITLE
20240606 minutes

### PR DESCRIPTION
**Pull request by planetf1**
TSC meeting minutes from 6 Jun 2024

### CODE REVIEW

File: meetings/2024-06-06/minutes.md [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+  * any noteable progress
+  * topics for next meeting
+* Any other business
+
+## Introductions / Welcome
+
+Apologies noted from Douglas, Matthias and Franziskus.
+
+JP Lomas introduced himself as a developer with QRL Foundation.
+Jonas Schneider-Bensch introduced himself as a developer from Crispin, and providing cover for Franziskus who can't make it today.
```
Comment (mkannwischer) [inline meetings/2024-06-06/minutes.md:39]:
```suggestion
Jonas Schneider-Bensch introduced himself as a developer from Cryspen, and providing cover for Franziskus who can't make it today.
```

File: meetings/2024-06-06/minutes.md [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+### What does assured mean
+
+* See [pq-code-package/documentation#8](https://github.com/pq-code-package/documentation/pull/8) for initial draft of docs
+* Hanno: need more refinement about formal proof. Add more links ie to aarch64 repo. Long term strive for functional correctness of the assembly. For example assembley level guarantees, constant time, trusted computing base assumptions.
+* Tiego - will look into discussion and see how he can contribute from the jasmine side, along with Manuel.
+
+### Common API
+
+* Tiago/ Hanno- general agreement with proposal. Funcion names should be namespaced (same signature) as multiple parts of code package could be used together
+* Should _derand functions be exposed to user? Diedre said that NIST explicitly stated redrand should not be part of full public api
```
Comment (mkannwischer) [inline meetings/2024-06-06/minutes.md:78]:
```suggestion
* Should _derand functions be exposed to user? Deirdre said that NIST explicitly stated redrand should not be part of full public api
```

Comment (planetf1) [reply meetings/2024-06-06/minutes.md:78]: Thanks for that correction. I thought I'd already spotted/corrected but it slipped through

**Comment by planetf1** [comment]
merging as no further comments

**Pull request merged**